include/flox/pkgdb/input.hh

```diff
@@ -274,7 +283,18 @@ struct PkgDbRegistryMixin : virtual public NixState {
     /** @return A list of systems to be scraped. */
     [[nodiscard]] virtual std::vector<std::string> & getSystems() = 0;
```

Comment:
TIL about `nodiscard`. What is setting this equal to 0 doing?

Reply:
it indicates that the function is "pure virtual", which means it is not implemented by _this_ class, and needs to be implemented in children.